src/module-elasticsuite-catalog-optimizer/Ui/Component/Listing/Column/BoostWeight.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade Smile ElasticSuite to newer
+ * versions in the future.
+ *
+ * @category  Smile
+ * @package   Smile\ElasticsuiteCatalogOptimizer
+ * @author    Pierre GAUTHIER <pigau@smile.fr>
+ * @copyright 2021 Smile
+ * @license   Open Software License ("OSL") v. 3.0
+ */
+namespace Smile\ElasticsuiteCatalogOptimizer\Ui\Component\Listing\Column;
+
+use Magento\Framework\Serialize\SerializerInterface;
+use Magento\Framework\View\Element\UiComponent\ContextInterface;
+use Magento\Framework\View\Element\UiComponentFactory;
+use Magento\Ui\Component\Listing\Columns\Column;
+
+/**
+ * Optimizers Boost Weight Column for Ui Component
+ *
+ * @category Smile
+ * @package  Smile\ElasticsuiteCatalogOptimizer
+ * @author   Pierre GAUTHIER <pigau@smile.fr>
+ */
+class BoostWeight extends Column
+{
+    /**
+     * Serializer
+     *
+     * @var SerializerInterface
+     */
+    protected $serializer;
+
+    /**
+     * @param ContextInterface $context
+     * @param UiComponentFactory $uiComponentFactory
+     * @param SerializerInterface $serializer
+     * @param array $components
+     * @param array $data
+     */
+    public function __construct(
+        ContextInterface $context,
+        UiComponentFactory $uiComponentFactory,
+        SerializerInterface $serializer,
+        array $components = [],
+        array $data = []
+    ) {
+        $this->serializer = $serializer;
+        parent::__construct($context, $uiComponentFactory, $components, $data);
+    }
+
+    /**
+     * Prepare Data Source
+     *
+     * @param array $dataSource
+     * @return array
+     */
+    public function prepareDataSource(array $dataSource)
+    {
+        if (isset($dataSource['data']['items'])) {
+            foreach ($dataSource['data']['items'] as & $item) {
+                $config = $this->serializer->unserialize($item['config']);
+                $item[$this->getData('name')] = $config['constant_score_value'] ?? '';
```

Comment:
Either there or somewhere else, I would add a "%" symbol, since we are displaying a score boost (50 = 50% boost of the score), and align the value to the right (I think it can be done through the listing XML).
See the explain module for reference.